docs/src/07_deployment_view.adoc

```diff
@@ -2,93 +2,39 @@ ifndef::imagesdir[:imagesdir: ../images]
 
 === Infrastructure Level 1
 
-[role="arc42help"]
-****
-Describe (usually in a combination of diagrams, tables, and text):
-
-* distribution of a system to multiple locations, environments, computers, processors, .., as well as physical connections between them
-* important justifications or motivations for this deployment structure
-* quality and/or performance features of this infrastructure
-* mapping of software artifacts to elements of this infrastructure
-
-For multiple environments or alternative deployments please copy and adapt this section of arc42 for all relevant environments.
-****
-image::DeploymentView.png[align="center",title="Deployment View",link="DeploymentView.png]
-_**<Overview Diagram>**_
-
 Motivation::
 
-_<explanation in text form>_
+The above diagram shows the initial version of the architecture and the division between its components. We will be using a simple client-server architecture, and the server will be querying Wikidata for the questions. By separating the architecture in this way we enforce a clear separation between the client/frontend and the server/backend, which benefits the system as a whole because so long the common API is implemented, the implementations themselves shall remain interchangeable.
 
 Quality and/or Performance Features::
 
-_<explanation in text form>_
+As stated before, we consider the main advantage of this architecture the interchangeability of the components. There remain details to be scratched out though, but this is still an initial version.
 
 Mapping of Building Blocks to Infrastructure::
-_<description of the mapping>_
 
+ * The webapp/frontend/client is contained within the `webapp` subfolder.
+ * The API/backend is contained within the `api` subfolder.
+ ** The database will be generated on deployment.
+ ** Wikidata is an external component, so although it is an important part of the deployment architecture, it is something we do not have access to changing.
 
 === Infrastructure Level 2
 
-[role="arc42help"]
-****
-Here you can include the internal structure of (some) infrastructure elements from level 1.
-
-Please copy the structure from level 1 for each selected element.
-****
-
-==== _<Infrastructure Element 1>_
+==== Frontend/Client
 
-_<diagram + explanation>_
+This component is self-contained and does not require much explanation.
 
-==== _<Infrastructure Element 2>_
+==== API/Backend
 
-_<diagram + explanation>_
+===== Server
 
-...
+Our main idea is that the server will be a self-contained .jar file with all the dependencies required within. It will communicate with the database and periodically query Wikidata to generate questions until a certain number has been obtained once per minute to fill up the database and improve speed.
 
-==== _<Infrastructure Element n>_
+===== Database
 
-_<diagram + explanation>_
+The database will contain the data used by the system. Therefore, it will contain user data and questions and their answers.
```

Comment:
As discussed on the last meeting on tuesday, we might wanna have different databases for questions and users, this might need to be mentioned.

Reply:
Forgot about that. I will update it right away.